docs/development/releasing.rst

```diff
@@ -175,26 +171,28 @@ packages that use the full bugfix/maintenance branch approach.)
 
 #. Push up all these changes to the `astropy core repository`_::
 
-      $ git push --tags upstream v<version>
+      $ git push --tags upstream v<version branch>.x
```

Comment:
``rename`` is definitely not intended...

To avoid this, I'll update the instructions to only push up the specific tag